src/python/platforms/android/adb.py

```diff
@@ -320,6 +320,10 @@ def is_gce():
   android_serial = environment.get_value('ANDROID_SERIAL')
   return android_serial.startswith('127.0.0.1:')
 
+def is_automotive():
+  """Returns if we are running in Android Automotive OS, currently only for Osprey."""
+  model = get_property('ro.product.model')
```

Comment:
Define get_property('ro.product.model') as a helper function in settings.py (get_product_model function)

Reply:
Thanks for pointing out, I think I can move is_automotive  to setting.py and use helper functions in settings.py directly